src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -17,12 +17,19 @@
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
 using Microsoft.ML.Runtime.TextAnalytics;
+using Microsoft.ML.Core.Data;
 
-[assembly: LoadableClass(typeof(LdaTransform), typeof(LdaTransform.Arguments), typeof(SignatureDataTransform),
-    LdaTransform.UserName, LdaTransform.LoaderSignature, LdaTransform.ShortName, DocName = "transform/LdaTransform.md")]
+[assembly: LoadableClass(LdaTransform.Summary, typeof(IDataTransform), typeof(LdaTransform), typeof(LdaTransform.Arguments), typeof(SignatureDataTransform),
+    "Latent Dirichlet Allocation Transform", "LdaTransform", "Lda")]
 
-[assembly: LoadableClass(typeof(LdaTransform), null, typeof(SignatureLoadDataTransform),
-    LdaTransform.UserName, LdaTransform.LoaderSignature)]
+[assembly: LoadableClass(LdaTransform.Summary, typeof(IDataTransform), typeof(LdaTransform), null, typeof(SignatureLoadDataTransform),
+    "Latent Dirichlet Allocation Transform", LdaTransform.LoaderSignature)]
+
+[assembly: LoadableClass(LdaTransform.Summary, typeof(LdaTransform), null, typeof(SignatureLoadModel),
+    "Latent Dirichlet Allocation Transform", LdaTransform.LoaderSignature)]
+
+[assembly: LoadableClass(typeof(IRowMapper), typeof(LdaTransform), null, typeof(SignatureLoadRowMapper),
+    "Latent Dirichlet Allocation Transform", LdaTransform.LoaderSignature)]
 
 namespace Microsoft.ML.Runtime.TextAnalytics
```

Comment:
>Runtime [](start = 23, length = 7)

Can you merge with master, I'm sure Senja changed this namespace to something better.